taskcluster/ci/ui-test/kind.yml

```diff
@@ -64,3 +64,31 @@ jobs:
                 - [automation/taskcluster/androidTest/ui-test.sh, x86-screenshots-tests, app.apk, android-test.apk, '-1']
         treeherder:
             symbol: debug(screenshots-x86)
+    x86-beta:
+        attributes:
+            build-type: debug
+            code-review: true
+            retrigger: true
+        dependencies:
+            signing: signing-beta-firebase
+            signing-android-test: signing-android-test-beta
+        run:
+            commands:
+                - [wget, {artifact-reference: '<signing-beta-firebase/public/build/x86/target.apk>'}, '-O', app.apk]
```

Comment:
@JohanLorenzo it is only sigining for debug, like the -debug is appended...but for beta we need to specify, would that be the way? I used the naming after the task name

Reply:
Also, are these wget commands overwritten too? If not I should take them out fo the defaults above...

Reply:
You can keep `<signing/public/build/x86/target.apk>` (and `<signing-android-test/public/build/noarch/target.apk>`). Taskgraph will know you refer to the beta tasks thanks to the mapping you provide in `dependencies` 

> Also, are these wget commands overwritten too? If not I should take them out fo the defaults above...

That's an excellent question! It's a behavior of taskgraph that I don't like. Taskgraph doesn't overwrite lists like `commands`. It concatenates them, instead:
 * https://hg.mozilla.org/ci/taskgraph/file/2b2622598df02bde211d8cedb334b7b22fb883a4/src/taskgraph/loader/transform.py#l47
 * https://hg.mozilla.org/ci/taskgraph/file/2b2622598df02bde211d8cedb334b7b22fb883a4/src/taskgraph/util/templates.py#l33

We need to take out `commands` from `job-defaults` :/

Reply:
Okay! thanks I will do that!